webapp/portlet/src/main/webapp/space-members/components/SpaceInvitationDrawer.vue

```diff
@@ -29,7 +31,38 @@
           {{ error }}
         </v-alert>
       </form>
+      <v-alert
+        v-else
+        class="ma-4"
+        outlined
+        text
+        color="green"
+      >
+        {{ this.$t('peopleList.label.successfulInvitation') }}
+      </v-alert>
+      <v-list v-if="externalUsers.length > 0" class="ma-4 rounded" style="background-color: #F6F8FA" subheader>
+        <v-list-item
+          v-for="user in externalUsers"
+          :key="user"
+        >
+          <v-list-item-avatar>
+            <v-img
+              :src="defaultAvatar"
+            ></v-img>
+          </v-list-item-avatar>
+          <v-list-item-content>
+            <v-list-item-title v-text="user"></v-list-item-title>
+          </v-list-item-content>
+
+          <v-btn icon @click="removeEternal(user)">
+            <v-icon color="#BE4141">
```

Comment:
can we avoid using hard coded color and use a css variable instead ?

CC @azayati